pkg/package_manager.go

```diff
@@ -196,11 +196,11 @@ func copyPackageAllFiles(targetTmpDir string, opts PackageCreateOpts) error {
 }
 
 type PackageCreateOpts struct {
-	PID            string
-	FilterRoots    []string
-	FilterFile     string
-	OnlyOneContent bool
```

Comment:
Confusing why it's here, I feel like some of the fields here are pretty internal and maybe they could be lowercase or in the separate non public struct